chunk/chunk.go

```diff
@@ -58,15 +58,18 @@ type ExecutableChunk struct {
 	// which is needed to correctly parse the end of the chunk.
 	BackQuotes int
 	Cfg        *config.Config
+	UI         view.UI
```

Comment:
The term UI seems a bit generic. What about RunnerView or any other combination with console, screen or terminal?